### ISSUE TITLE
<span> addon not the same height as input on Firefox (32.0.1)

### ISSUE THREAD

**Issue by klingtnet** [owenversteeg/min#43]
See the screenshot below:
![minfwk_bug_span_firefox](https://cloud.githubusercontent.com/assets/1538057/4290996/99ed3898-3dc0-11e4-971a-c9332b6a0535.png)

**Comment by owenversteeg** [comment]
Using my magic psychic powers I will guess that either you have customized font settings or you are on Linux, and most likely not using a popular distro either (I've tested this on popular distros.) :smile: 

In all seriousness: What OS are you using, and do you have any customized font settings? Thanks!

**Comment by klingtnet** [comment]
Your magic psychic powers are working :smiley:
It's Firefox running on 64bit Arch Linux (kernel 3.16.2-1, Gnome 3.12) without customized font settings. Checked it on a different machine running the same os and browser version resulting in the same height difference.

**Comment by owenversteeg** [comment]
Yep, Arch has problems with addons due to it not coming with a good set of sans-serif fonts. I use Arch myself, and problems with Arch and Chromium involving addons are documented in #23. Given the extremely small number of people using Arch, and given that most Arch installs are vastly different from each other, I think that this is something that can be postponed, or in the worst case ignored. There's a relatively small fix for Chromium, documented in #23, but I'm not sure what I can do to fix Firefox. Finally, of the Arch users I know three of four don't even see this issue because they're using Nimbus Sans or an equivalent as the default sans-serif font.

tl;dr This affects a small portion of users running Arch Linux, and there is no small-size fix; thus, a fix is unlikely.

**Comment by klingtnet** [comment]
I've read issue [#23](https://github.com/owenversteeg/min/issues/23) and thank your time spend investigating this issue. Even if this affects only a very small portion of users, I've done the following modification to fix this issue on my system:
Increasing the `font-size` of `textarea,input,.addon,.btn-sm,.nav a` from `14px` to `16px` fixes the problem in Firefox, while still looking nice on chromium, but slightly increases the input element size.

<del>Maybe you could check the compatibility of this modification with other browser, system combinations and make it the default if you like.</del>

Sadly this looks broken on other browser/OS combinations.

**Comment by bnb** [comment]
I'm on Win8/FF latest with no custom font settings, and I have the same problem. Noticed immediately when I went to the docs.

**Comment by owenversteeg** [comment]
I am sadly marking this as wontfix, as this only affects a portion of users running Arch Linux, which is an extremely small number of people.

@bnb I can't reproduce this on my Windows/Firefox combos so I'm going to guess that your machine is somehow customized in a way that causes this bug. Again, no known fix and affects a very small portion of users.
